app/src/main/res/layout/item_contributor.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="contributor"
+            type="io.github.droidkaigi.confsched.model.Contributor" />
+    </data>
+
+    <android.support.v7.widget.CardView
```

Comment:
I guess it is better not to use CardView here.

<img width="807" alt="cards_-_components_-_google_design_guidelines" src="https://cloud.githubusercontent.com/assets/1269214/13028026/4035670e-d2a6-11e5-8955-7ab56f29a6ac.png">

https://www.google.com/design/spec/components/cards.html#cards-usage

But this is design matter! You don't need to fix it :+1: